build.gradle

```diff
@@ -143,7 +143,7 @@ dependencies {
     compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-openfeign', version: '2.2.7.RELEASE'
 
     compile group: 'com.github.hmcts.java-logging', name: 'logging', version: reformLogging
-    compile group: 'com.github.hmcts.java-logging', name: 'logging-appinsights', version: reformLogging
+    compile group: 'com.microsoft.azure', name: 'applicationinsights-spring-boot-starter', version: '2.6.1'
```

Comment:
Do we need a replacement at all? The latest Reform template only includes `com.github.hmcts.java-logging.logging` - https://github.com/hmcts/spring-boot-template/blob/77af92e0748af0c56062f9bb2c05d84e2a9886e7/build.gradle#L192

Reply:
If we don't include applicationinsights-spring-boot-starter then we are getting some compilation errors as TelemetryClientclass not found.